packages/terra-doc-template/docs/README.md

```diff
@@ -0,0 +1,18 @@
+# Terra Doc Template
+
+Provides an easily adjustable template for documentation pages
```

Comment:
I would keep this description simple;

`Provitdes a template for documentation`